app/upgrades/v2/upgrades_test.go

```diff
@@ -0,0 +1,92 @@
+package v2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/tharsis/evmos/v2/app"
+	claimsMigrations "github.com/tharsis/evmos/v2/x/claims/migrations/v2"
+	erc20Migrations "github.com/tharsis/evmos/v2/x/erc20/migrations/v2"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	feemarkettypes "github.com/tharsis/ethermint/x/feemarket/types"
+)
+
+type UpgradeTestSuite struct {
+	suite.Suite
+
+	ctx sdk.Context
+	app *app.Evmos
+}
+
+func (suite *UpgradeTestSuite) SetupTest() {
+	// consensus key
+	// consAddress := sdk.ConsAddress(tests.GenerateAddress().Bytes())
+
+	// setup feemarketGenesis params
+	feemarketGenesis := feemarkettypes.DefaultGenesisState()
+	feemarketGenesis.Params.EnableHeight = 1
+	feemarketGenesis.Params.NoBaseFee = false
+
+	suite.app = app.Setup(false, feemarketGenesis)
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{
+		ChainID: "evmos_9001-2",
+	})
+}
+
+func TestUpgradeTestSuite(t *testing.T) {
+	suite.Run(t, new(UpgradeTestSuite))
+}
+
+func (suite *UpgradeTestSuite) TestUpdateEVMHooks() {
```

Comment:
Probably should call this something else since it's:
1. Updating params on the ERC20 module
2. Updating params on the claims module
3. EVM hooks are not summative of the migration entirely

Reply:
,,,I think now that it actually goes through the forks logic (instead of setting params from migration.go earlier) the name should be fine, wdyt?